app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -1,5 +1,5 @@
 module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
-  include ActiveSupport::Concern
+  extend ActiveSupport::Concern
```

Comment:
this is a tangent.
I think the existing one is a bug but can pull into another PR if you want